daml-assistant/daml-helper/src/DamlHelper/Run.hs

```diff
@@ -665,18 +669,43 @@ runStart (StartNavigator shouldStartNavigator) (OpenBrowser shouldOpenBrowser) o
             when shouldWaitForSignal $
                 void $ race (waitForProcess navigatorPh) (waitForProcess sandboxPh)
 
-    where sandboxPort = SandboxPort 6865
-          navigatorPort = NavigatorPort 7500
+    where navigatorPort = NavigatorPort 7500
           withNavigator' sandboxPh =
               if shouldStartNavigator
                   then withNavigator
                   else (\_ _ _ f -> f sandboxPh)
 
+runDeploy :: IO ()
+runDeploy = do
+    projectName <- getProjectName
+    let darPath = ".daml" </> "dist" </> projectName <> ".dar"
+    assistant <- getDamlAssistant
+    callCommand (unwords $ assistant : ["build"])
+    let SandboxPort port = sandboxPort
```

Comment:
We should at least make the port configurable for now given that `daml sandbox` also allows you to change that quite easily. Ofc eventually, we need this to be fully configurable including the host, SSL config, … but for an initial version the port is good enough imho.

Reply:
done!
made sandbox port configurable for deploy (and also start command)